docs-src/concepts/what-is-a-schedule.md

```diff
@@ -201,6 +201,15 @@ It is important to note that the [status](/concepts/what-is-a-workflow-execution
 
 :::
 
+:::caution
+
+A scheduled workflow may complete with a result up to the maximum blob size (2 MiB by default).
+However, due to internal limitations, results that are within 1 KiB of this limit cannot be passed to the next execution.
+So, for example, a workflow that returns a result of size above 2,096,127 (2MiB - 1KiB) bytes will be allowed to compete successfully, but that value will not be available as a last completion result.
```

Comment:
Reading strictly, this says that a workflow that returns any size above 2,096,127 will be allowed to complete successfully. That's not true, it has to be <= 2097152. That's why I wrote it with a specific number as an example. Written this way, it's no longer an example, it's just a statement.

Either make it use an example (I think this is cleanest and prefer it), or else reword it to be strictly correct with the min and max and remove mention of "example"

Reply:
Also, please capitalize core terms like "Workflow".
And specify "Workflow Execution".